scripts/lib/docker-utils.sh

```
@@ -437,6 +437,97 @@ test_chat_endpoint() {
+        log_warning \
+            "Skipping MCP live-data smoke check because ENABLE_BISQ_MCP_INTEGRATION is disabled"
+        return 0
+    fi
+
+    payload=$(jq -nc --arg question "$question" \
+        '{question: $question, chat_history: [], bypass_hooks: ["escalation"]}')
+
+    log_info "Testing MCP live-data chat endpoint..."
+
+    local attempt=1
+    while [ $attempt -le $retries ]; do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)